l10n_ch_pain_direct_debit/__manifest__.py

```diff
@@ -1,17 +1,29 @@
-# -*- coding: utf-8 -*-
 # © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
+# pylint: disable=C8101
```

Comment:
Weird, C8101 is a check about presence of Odoo Community Association (OCA) in the author list.

My guess is that by removing the space between `Association` and `(OCA)` you get a lint error. 

This is the reference:

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py